examples/basic.rs

```diff
@@ -331,9 +579,46 @@ async fn signed_unencrypted() {
         &msg,
         &did_resolver,
         &secrets_resolver,
-        &UnpackOptions {
-            ..UnpackOptions::default()
-        },
+        &UnpackOptions::default(),
+    )
+    .await
+    .expect("Unable unpack");
+
+    println!("Received message is \n{:?}\n", msg);
+    println!("Received message unpack metadata is \n{:?}\n", metadata);
+}
+
+async fn plaintext_unencrypted() {
+    // --- Building message from ALICE to BOB ---
+    let msg = Message::build(
+        "example-1".to_owned(),
+        "example/v1".to_owned(),
+        json!("example-body"),
+    )
+    .to(BOB_DID.to_owned())
+    .from(ALICE_DID.to_owned())
+    .finalize();
+
+    // --- Packing plaintext message ---
+    let did_resolver = ExampleDIDResolver::new(vec![ALICE_DID_DOC.clone(), BOB_DID_DOC.clone()]);
+
+    let msg = msg
+        .pack_plaintext(&did_resolver)
+        .await
+        .expect("Unable pack_plaintext");
+
+    // --- Sending message ---
+    println!("Sending message \n{}\n", msg);
+
+    // --- Unpacking message ---
+    let did_resolver = ExampleDIDResolver::new(vec![ALICE_DID_DOC.clone(), BOB_DID_DOC.clone()]);
+    let secrets_resolver = ExampleSecretsResolver::new(BOB_SECRETS.clone());
```

Comment:
Maybe keep secrets_resolver empty here to avoid confusion?

Reply:
Done.